src/test/java/ch/poole/openinghoursparser/IndividualTest.java

```diff
@@ -0,0 +1,54 @@
+package ch.poole.openinghoursparser;
+
+import java.io.ByteArrayInputStream;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Individual testing for the OpeningHoursParser, receiving
+ * inputs from System.in
+ * 
+ * 
+ * @author Vuong Ho
+ *
+ */
+
+public class IndividualTest {
+    
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in); 
+        
+        Boolean isStrict = (args[0].equals("true")) ? true : false;
+
+        while(true) {
+            System.out.print("Please enter your input: ");
+            String input = sc.nextLine();
+
+            try {
+                OpeningHoursParser parser = new OpeningHoursParser(new ByteArrayInputStream(input.getBytes()));
+                List<ch.poole.openinghoursparser.Rule> rules = parser.rules(isStrict);
+                
+                System.out.println("Legal input string");
+                System.out.println("Detected rules in input string listed below");
+                for(ch.poole.openinghoursparser.Rule rule : rules) {
+                    System.out.println(rule.toDebugString());
+                }
+                System.out.println();
+                System.out.println("------------------------------");
+                System.out.println();
+    
+            } catch (OpeningHoursParseException e) {
+                System.out.println("Illegal input string");
+                e.printStackTrace();
+                System.out.println();
+                System.out.println("------------------------------");
+                System.out.println();
+            }
+        }
+        
```

Comment:
Lots of gratuitous empty lines here, for what every reason I find them visually extremely annoying between braces, could you remove them, thanks.